redaxo/src/addons/mediapool/assets/mediapool.js

```diff
@@ -83,8 +83,13 @@ function viewREXMedia(id,param)
 
 function deleteREXMedia(id)
 {
-    var a = new getObj("REX_MEDIA_"+id);
-    a.obj.value = "";
+    var input = new getObj("REX_MEDIA_" + id).obj;
+    if (input !== null) {
+        input.value = "";
+        jQuery("#REX_MEDIA_" + id).trigger('change');
```

Comment:
```suggestion
        jQuery(input).trigger('change');
```

So müsste es dann eigentlich auch gehen.
Allgemein, ob die Variante über jQuery jetzt besser ist, ehrlich gesagt keine Ahnung.
Für mich aber ok so, denke ich.

Reply:
Mit jQuery funktioniert es im Firefox, Chrome und IE11.